wagtail/contrib/wagtailstyleguide/views.py

```diff
@@ -2,15 +2,24 @@
 
 from django import forms
 from django.core.paginator import Paginator
+from django.db import models
 from django.shortcuts import render
 from django.utils.translation import ugettext as _
-
 from wagtail.wagtailadmin import messages
 from wagtail.wagtailadmin.forms import SearchForm
 from wagtail.wagtailadmin.widgets import (
-    AdminDateInput, AdminDateTimeInput, AdminPageChooser, AdminTimeInput)
+    AdminAutoHeightTextInput, AdminDateInput, AdminDateTimeInput, AdminPageChooser, AdminTimeInput)
+from wagtail.wagtailcore.fields import RichTextArea
 from wagtail.wagtaildocs.widgets import AdminDocumentChooser
 from wagtail.wagtailimages.widgets import AdminImageChooser
+from wagtail.wagtailsnippets.widgets import AdminSnippetChooser
+
+
+class ExampleSnippet(models.Model):
```

Comment:
This model fails tests. We need to create migration or mock / fake class with `_meta.verbose_name`, `_meta.app_label`. etc.